ui/pages/send/send.test.js

```diff
@@ -247,6 +247,7 @@ describe('Send Page', () => {
             id: 3111025347726181,
             time: 1620723786838,
             status: 'unapproved',
+            metamaskNetworkId: '5',
```

Comment:
This seems like a odd addition for this PR, were you fixing something unrelated?

Reply:
This is an old field, it can be removed now. Changed in 47fd68abbcc340838a25f1a0619cb06d04ceb83b